fineract-provider/src/main/resources/db/changelog/custom-changelog/CI18-92_add_undo_reschedule_permission.xml

```diff
@@ -22,13 +22,14 @@
 <databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
                    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
                    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.1.xsd">
-    <changeSet author="judio@fiter.io" id="CI18-92_add_undo_reschedule_permission">
+    <changeSet author="judio@fiter.io" id="CI18-92_add_undo_reschedule_permission01">
         <preConditions onFail="MARK_RAN">
             <sqlCheck expectedResult="0">
                 SELECT COUNT(1) FROM m_permission WHERE code = 'UNDO_RESCHEDULELOAN'
             </sqlCheck>
         </preConditions>
         <insert tableName="m_permission">
+            <column name="id" value="10092"/>
```

Comment:
is there a reason why the id has to be specified explicitly, i see this causing issues with other deployments

Reply:
@Napho this is due to postgres auto increment issues. It also happens when you run all scripts on an empty database. We'll look into the root issue and avoid these going forward. For now we need to merge these commits into carbon branch to unblock the team.